docs/content/manpages/geowave-dbscan.txt

```diff
@@ -0,0 +1,144 @@
+//:geowave-dbscan(1)
+//:======================
+//::doctype: manpage
+
+NAME
+//:----
+
+geowave analytic dbscan - Density Based Scanner
+
+SYNOPSIS
+//:--------
```

Comment:
Add comments below.
Analytic 'nn' is similar to DBScan, with less arguments.  
'nn' just dumps all near neighbors for every feature id in a list of pairs.
Most developers will want to extend the framework to add their own extensions.